android/titanium/src/java/org/appcelerator/titanium/TiApplication.java

```diff
@@ -769,6 +769,32 @@ public boolean isCoverageEnabled()
 		return false;
 	}
 
+	public static void launch()
+	{
+		final TiRootActivity rootActivity = (TiRootActivity) TiApplication.getAppRootOrCurrentActivity();
```

Comment:
We need to call `getRootActivity()` instead.
Calling `getAppRootOrCurrentActivity()` can return a `TiActivity` object when running in LiveView.

We may want to check if `rootActivity` is `null` just in case Android developer option "Don't keep activities" is enabled... although our `softRestart()` method checks for this already.